portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreation.testcase

```diff
@@ -944,8 +952,12 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8479 | Test Summary: Create a Segment with a Number Property"
+	@ignore = "true"
 	@priority = "4"
 	test CanCreateSegmentWithNumberProperty {
+
+		// CSV data source tests are no longer possible to run due to this ticket LRAC-11651
+
```

Comment:
Hey @manuferreira, I think this validation can be done using other criteria.

I think you can undo this change and create a ticket for us to refactor this test or you can do it yourself in your ticket

Reply:
This ticket is probable to have problems with automation because of https://issues.liferay.com/browse/LRAC-9329

Reply:
This bug that you pointed out is a static segment that is being created and later on a distribution filter and the test that I'm editing now `CanCreateSegmentWithNumberProperty` creates only a dynamic segment with an age field. So I don't think this test will have issues, or am I missing something? what do you think?

Reply:
Ohh yes, I think this bug won't be a problem in this test. Maybe the delay of the membership list only